languages/js/sdk-client/src/client.ts

```diff
@@ -131,6 +125,30 @@ export class SecretsClient {
 
     return handleResponse(Convert.toResponseForSecretsDeleteResponse(response));
   }
+
+  async sync(organizationId: string, lastSyncedDate: Date): Promise<SecretsSyncResponse> {
```

Comment:
```suggestion
  async sync(organizationId: string, lastSyncedDate?: Date): Promise<SecretsSyncResponse> {
```

I believe passing the `lastSyncedDate` is optional.

Reply:
Good call, resolved in https://github.com/bitwarden/sdk/pull/1044/commits/7f576ceabf2ecd736d2b57d8b8e2fabed46ee769